core/templates/dev/head/pages/collection_player/collection_footer_directive.html

```diff
@@ -0,0 +1,161 @@
+<script type="text/ng-template" id="components/collectionFooter">
+  <div class="footer navbar-fixed-bottom oppia-collection-footer">
+    <div class="row">
+      <div class="col-sm-5">
+        <ul class="author-profile">
```

Comment:
I essentially followed the styling used in the footer for the exploration_player so didn't change the names. I'll change the names now.